include/ck/tensor_operation/gpu/grid/gridwise_binary_elementwise_1d.hpp

```diff
@@ -34,29 +36,34 @@ template <typename ADataType,
           typename BDataType,
           typename CDataType,
           typename ComputeDataType,
-          typename GridDesc_M0,
+          typename AGridDesc_M0,
+          typename BGridDesc_M0,
+          typename CGridDesc_M0,
           typename ElementwiseFunctor,
-          index_t ScalarPerVector>
+          index_t M0PerThread,
```

Comment:
Again, wondering why ```M0``` instead of simply ```M```

Reply:
221146a0d0019dad0f0641678948ec3763ceddf7